src/Symfony/Component/Mailer/Transport/Smtp/SmtpTransport.php

```diff
@@ -179,6 +207,8 @@ protected function doSend(SentMessage $message): void
             $this->stream->flush();
             $this->executeCommand("\r\n.\r\n", [250]);
             $message->appendDebug($this->stream->getDebug());
+
+            $this->lastSent = microtime(true);
```

Comment:
You only update lastSent when a message has been sent. In case of an error, it won't be reset. By reading the issue, I would have expected that lastSent would be the last time we sent a ping.

Reply:
Yes, because the variable should represent the time when the last message was sent. I chose the name to be consistent with the parent `AbstractTransport`. Do want me to change it?

Reply:
But if sending the email does not work for some reason, you will ping the server next time you try, not sure if that's what we want.

Reply:
I don't have a strong opinion about this, so I changed it. I updated the name of the field as well, since it now has a slightly different meaning then the one in the `AbstractTransport`.